Core/GDCore/Project/Project.h

```diff
@@ -890,6 +903,7 @@ class GD_CORE_API Project : public ObjectsContainer {
                         ///< are below this number )
   bool verticalSync;    ///< If true, must activate vertical synchronization.
   gd::String scaleMode;
+  bool pixelsRounding; ///< If true, set PIXI's ROUND_PIXELS option to true

```

Comment:
Nitpicking: In the `Project` and in `Core` in general, we usually avoid mentioning PIXI. The idea is that the `Core` describe a game (project, scenes, events, etc...) and tools to manipulate it (for example, a class like `WholeProjectRefactorer` that allows you to rename an object in the whole project after the user rename it in a layout, this kind of "quality of life" stuff). `GDJS` uses `Core` to implement JavaScript specific stuff (code generation mainly! And setting the function names).
And finally, the `GDJS` game engine is the one implementing the runtime of a game that was code generated. In all of this PixiJS is... almost an implementation detail :) Well not really in GDJS (we know that we must copy the PixiJS files and renderer files), but in `Core` it should be agnostic of it.

This could be useful should we in the future choose to:
- build another game engine not based on JS (unlikely though in the short term, too much duplicated work, not super agile).
- do rendering using something else than PixiJS. While this can be unlikely for games, there is I think a real possibility that we in the future create a "empty renderer" or "memory renderer" that render games on... nothing. To be able to run a game on something like a Node.js headless server (useful if you want to make a multiplayer game server).

All of this to say, something like "If true, the rendering should stop pixel interpolation of rendered objects." ;)